pandas/core/algorithms.py

```diff
@@ -903,7 +904,8 @@ def value_counts_arraylike(values, dropna: bool):
 
 
 def duplicated(
-    values: ArrayLike, keep: Literal["first", "last", False] = "first"
+    values: np.ndarray | ExtensionArray | Series,
```

Comment:
`ArrayLike` is a alias for `np.ndarray | ExtensionArray` and can union aliases with other types

```suggestion
    values: ArrayLike | Series,
```

---

`AnyArrayLike` also includes Index (as well as Series). Is an Index not a valid type here?

Reply:
@simonjayhawkins 
Thanks for the review! 
> AnyArrayLike also includes Index (as well as Series). Is an Index not a valid type here?
I checked index type and it seems duplicated function accepts one. 
```
import pandas as pd
from pandas.core.algorithms import duplicated
index = pd.Index([1,2,3,4,2,3,1])
duplicated(index)

>> array([False, False, False, False,  True,  True,  True])
```
Maybe it will be like this? 
```suggestion
    values: AnyArrayLike,
```

Reply:
Great. can you also update the parameters section of the docsting.

Reply:
Yes! I have pushed the code with those change as well!